tests/unit/test_bundle.py

```diff
@@ -246,19 +253,21 @@ async def test_run_local(self):
                                          constraints="constraints",
                                          endpoint_bindings="endpoint_bindings",
                                          resources={},
-                                         storage="storage",
+                                         storage={storage_label: constraints.parse_storage_constraint(storage_constraint)},
                                          devices="devices",
                                          num_units="num_units",
                                          channel="",
                                          charm_origin=ANY)
 
     async def test_run_no_series(self):
+        storage_label = "some-label"
+        storage_constraint = "ebs,100G,1"
```

Comment:
Can we increase test coverage to include storage for more than one named storage, ie not just a single "storage-label" but a couple of storages like "data" and "logs" or something.